def k(_):
    raise Exception()

sorted([42], key=k)